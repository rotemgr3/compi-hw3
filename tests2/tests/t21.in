 
//piazza 88 - should be ok +
override int foo(int y) {int booga = 5;}
override int foo(int y, bool x) {int booga = 5;}
void main() {int booga = 5;}
