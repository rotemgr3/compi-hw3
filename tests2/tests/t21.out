---end scope---
y INT -1
booga INT 0
---end scope---
y INT -1
x BOOL -2
booga INT 0
---end scope---
booga INT 0
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
foo (INT)->INT 0
foo (INT,BOOL)->INT 0
main ()->VOID 0
